File: PROJECT/FRONTEND/Common/DTOs/OrderDetailsDTO.ts


export class OrderDetailsDTO { 
    constructor(obj: Partial<OrderDetailsDTO>) {
        Object.assign(this,obj);
    }

    
    public id: number;
    public clientFirstName: string;
    public clientLastName: string;
    public providedAddress: string;
    public address: IAddressDTO;
}

File: PROJECT/Models/DTOs/Internal/Orders/OrderDetailsDTO.cs
using Models.DTOs.Internal.Products;
using Models.DTOs.Shipping;

namespace Models.DTOs.Internal.Orders
{
    public class OrderDetailsDTO
    {
        public int Id { get; set; }
        public string ClientFirstName { get; set; }
        public string ClientLastName { get; set; }
        public string ProvidedAddress { get; set; }
        public IAddressDTO Address { get; set; }
        public List<ProductItemDTO> Products { get; set; }
    }
}
